onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -219,10 +219,13 @@ def optimize_model(
     if opt_level is None:
         opt_level = default_opt_level
 
+    # Disable constant sharing to avoid model proto str test mismatch. Ideally the optmizer should not
+    # affect other fusions. We can update the expected model_proto once the optimizer logic becomes stable.
```

Comment:
nit: optmizer -> optimizer.

Maybe clarify to 'once the ConstantSharing optimizer logic becomes stable'